Makefile

```diff
@@ -22,6 +22,13 @@ BUILD_FLAGS = "-X '$(REPO)/pkg/config.Version=$(VERSION)' -X '$(REPO)/cli/smartc
 
 IMAGE_REPO=nspccdev/neo-go
 
+# .deb package versioning
+OS_RELEASE = $(shell lsb_release -cs)
+PKG_VERSION ?= $(shell echo $(VERSION) | sed "s/^v//" | \
+			sed -E "s/(.*)-(g[a-fA-F0-9]{6,8})(.*)/\1\3~\2/" | \
```

Comment:
Why this one?

Reply:
It allows to include git hash aside from tag to allow developers builds be easily differentiated and sorted via apt.
Alas, Debian rules for package versioning are pretty restricted so various "hacks" are used here.